meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging-mfgtool_2023.04.bb

```diff
@@ -0,0 +1,10 @@
+SUMMARY = "Produces a Manufacturing Tool compatible U-Boot"
+DESCRIPTION = "U-Boot recipe that produces a Manufacturing Tool compatible \
+binary to be used in updater environment"
+
+require recipes-bsp/u-boot/u-boot-ti-staging_2023.04.bb
```

Comment:
better to not hardcode the version and use the PV so it will on next updates just renaming the recipe
```
-- require recipes-bsp/u-boot/u-boot-ti-staging_2023.04.bb
++ require recipes-bsp/u-boot/u-boot-ti-staging_${PV}.bb
```

Reply:
fixed